test/wallet-coinselection-test.js

```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+const assert = require('bsert');
+const {
+  MTX,
+  Network,
+  WalletDB
+} = require('..');
+
+// Use main instead of regtest because (deprecated)
+// CoinSelector.MAX_FEE was network agnostic
+const network = Network.get('main');
```

Comment:
Configurable policy could help here as well.

Reply:
I think I'm going to keep this on main. Configurable absurd factor doesnt matter between networks, but we have default fee rate different for each network. 

What I think is important is this proof that the highest possible fee at default rate is 10 HNS:

https://github.com/pinheadmz/hsd/blob/48d93c452020562807b550119f96425b2eafa19c/test/wallet-coinselection-test.js#L91-L99